schemas/student.py

```diff
@@ -23,49 +24,47 @@ class CreateStudentIn(BaseModel):
     gender: str
 
     @validator('full_name')
-    def validate_full_name(value):
-        full_name = value.split()
-        if not full_name:
-            raise ValueError("Прізвище та ім'я студента обов'язкові до заповнення!")
-        elif len(full_name) < 2:
-            raise ValueError("Прізвище та ім'я студента обов'язкові до заповнення!")
+    def validate_full_name(value):  # TODO may be there is need to use staticmethod decorator
+        full_name = value.split()   # TODO use hint or so
+        if not full_name or len(full_name) < 2:
+            raise BackendException(message="The student's name and surname are mandatory!")
         return value
     
     @validator('telephone_number')
-    def validate_telephone_number(value):
+    def validate_telephone_number(value):   # TODO may be there is need to use staticmethod decorator
         if not value:
-            raise ValueError('Телефонний номер не може бути порожнім!')
+            raise BackendException(message='The phone number cannot be empty!')
         elif len(str(value)) != 12:
-            raise ValueError('Телефонний номер має містити в собі 12 цифр!')
+            raise BackendException(message='The phone number must contain 12 digits!')
         return value
 
     @validator('course_id')
-    def validate_course_id(value):
+    def validate_course_id(value):  # TODO may be there is need to use staticmethod decorator
         if not value:
-            raise ValueError('Курс не може бути порожнім!')
+            raise BackendException(message='The course cannot be empty!')
         elif value not in range(1, 7):
-            raise ValueError('Курс моє бути між 1 та 6!')
+            raise BackendException(message='The course must be between 1 and 6!')
         return value
 
     @validator('speciality_id')
     def validate_speciality_id(value):
         if not value:
-            raise ValueError('Cпеціальність не може бути порожньою!')
+            raise BackendException(message='The specialty cannot be empty!')
```

Comment:
Don't use `BackendException` inside Pydantic schemas validation. it should be handled by another handler: `validation_exception_handler`.